### ISSUE TITLE
Enhancement - Node IPs of cluster to be visible in cluster management UI not present in the API

### ISSUE THREAD

**Issue by anupama2501** [rancher/rancher#32759]
**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Steps to reproduce (least amount of steps as possible):**

- Create a cluster with 1 node, all roles.
- Once the cluster is active, nodes can be viewed/accessed from EXPLORE CLUSTER --> select the cluster --> cluster dropdown nodes.

**Result:**

- enhancement for the nodes' IPs to be displayed from the cluster management --> cluster page for easier access
- Backend issue as for the UI to display, feature is not present in the API

**Refer:** https://github.com/rancher/dashboard/issues/2957

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): master-head 5d2b7ac
- Installation option (single install/HA): Single

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Node driver

**Comment by jiaqiluo** [comment]
It turns out the information is in the v1 API but under the status field in the endpoint `v1/management.cattle.io.nodes/<cluster-name>/<node-name>`. 
UI should look up the value following some logic like this
```
if t == "InternalIP" {
data["ipAddress"] = a
} else if t == "ExternalIP" {
data[extIPField] = a
} else if t == "Hostname" {
data["hostname"] = a
}
```

**Comment by gaktive** [comment]
The related backend work in https://github.com/rancher/dashboard/issues/2957 is now merged for 2.6.7 so UI can unblock work on this now. Removing the `awaiting-ui` label.

**Comment by snasovich** [comment]
@gaktive , my understanding this was done as part of https://github.com/rancher/dashboard/issues/2957 and there is no back-end work left to do so closing this issue. Feel free to reopen if it's not the case.
